tests/utils.go

```diff
@@ -1694,8 +1694,9 @@ func removeNamespaces() {
 	fmt.Println("")
 	for _, namespace := range testNamespaces {
 		fmt.Printf("Waiting for namespace %s to be removed, this can take a while ...\n", namespace)
-		EventuallyWithOffset(1, func() bool { return errors.IsNotFound(virtCli.CoreV1().Namespaces().Delete(namespace, nil)) }, 240*time.Second, 1*time.Second).
-			Should(BeTrue())
+		EventuallyWithOffset(1, func() bool {
+			return errors.IsNotFound(virtCli.CoreV1().Namespaces().Delete(namespace, nil))
```

Comment:
same here https://github.com/kubevirt/kubevirt/pull/3797#discussion_r472232618

Reply:
Done

Reply:
I do not see this change, is it waiting for you to push it?

Reply:
Sorry I missed that I will push it right away